src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+export interface ApiKeyModel {
+    organizationId?: string;
+    id?: string;
+    enabled?: boolean;
+    value?: string;
+    displayName?: string;
+    description?: string;
+    createdBy?: {};
+    createdDate?: number;
+    allowedIps?: string[];
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+    deniedIps?: string[];
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    privileges?: PrivilegeModel[];
+    resourceId?: string;
+}
+
+export enum Provider {

```

Comment:
This enum is already defined in the GroupInterfaces file, I think It should be shared between both resources.

Reply:
Yes, but in the enum AuthProvider of GroupsInterface.ts, we don't have `OTHER = 'OTHER',` 
Also, should we move it to BaseInterfaces.ts?

Reply:
See [Jack's comment](https://github.com/coveo/platform-client/pull/14#discussion_r330627573) about the `OTHER` enum value.

Yes I think that would be the place to put interfaces that are shared between resources :+1: